app/code/Magento/Downloadable/Observer/SaveDownloadableOrderItemObserver.php

```diff
@@ -131,6 +131,13 @@ public function execute(\Magento\Framework\Event\Observer $observer)
                         ScopeInterface::SCOPE_STORE
                     );
                 $linkPurchased->setLinkSectionTitle($linkSectionTitle)->save();
+                
+                $linkstatus =  \Magento\Downloadable\Model\Link\Purchased\Item::LINK_STATUS_PENDING;
+               if( $orderStatusToEnableItem == \Magento\Sales\Model\Order\Item::STATUS_PENDING
```

Comment:
Please fix minor styling issues for `if` statement according to PSR-2 standard (spacings etc.).
An `if `structure should look like the following:
```
if ($a == $b
    || $b == $c
    || $c == $d
) {
    // code here
}
```

Reply:
sure